Source/JavaScriptCore/runtime/Structure.cpp

```diff
@@ -52,7 +52,7 @@ inline void StructureTransitionTable::setSingleTransition(VM& vm, JSCell* owner,
     vm.writeBarrier(owner, structure);
 }
 
-bool StructureTransitionTable::contains(UniquedStringImpl* rep, unsigned attributes, TransitionKind transitionKind) const
+bool StructureTransitionTable::contains(void* rep, unsigned attributes, TransitionKind transitionKind) const
```

Comment:
I think it would be better to have a union type here for readability.

Reply:
Done.